indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -825,6 +831,9 @@ class BookingDetails extends React.Component {
     const showActionButtons = !isCancelled && !isRejected && (canCancel || canReject || showAccept);
     const activeBookings = _.omitBy(occurrences.bookings, value => _.isEmpty(value));
     const occurrenceCount = Object.keys(activeBookings).length;
+    const cancellableOccurrenceCount = Object.keys(activeBookings).filter(date =>
```

Comment:
`Object.entries()` is the better choice here since you need the values as well. Or `Object.values()` since you don't use the keys except to get the values..